src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -12,8 +12,25 @@
                 i18n>Grafana Dashboard doesn't exist. Please refer to
   <cd-doc section="grafana"></cd-doc> on how to add dashboards to Grafana.</cd-alert-panel>
 
+  <cd-alert-panel type="info"
+                  *ngIf="!panelLoaded && !isChrome"
+                  i18n>The embedded Grafana Dashboard was blocked due to a certificate error, please click <a [href]="grafanaSrc"
+                     target="_blank"
+                     noopener
+                     noreferrer>here</a> to accept the security warning then reload this page</cd-alert-panel>
+
+
+  <cd-alert-panel type="info"
+                  *ngIf="isChrome"
```

Comment:
```suggestion
                  *ngIf="isChrome && !panelLoaded"
```

We need this right? Otherwise even if we accept the cookies this alert won't go away. 

![Screenshot from 2022-04-25 22-33-06](https://user-images.githubusercontent.com/71764184/165138113-51c983a2-5b3b-43ef-8629-f36c4c50acf7.png)

Also doing this will basically introduce this [ExpressionChangedAfterItHasBeenCheckedError](https://angular.io/errors/NG0100) which I think can be bypassed with a force change detection on the onLoad() function like this

```
onLoad() {
    this.panelLoaded = true;
    this.cdRef.detectChanges();
  }
```
(ofcourse you'll need to initialize `private cdRef: ChangeDetectorRef` in constructor.

@epuertat What's your take on it? Given the effort @nSedrickm put here I think its something we can afford.